frontend/src/pages/org/collection-detail.ts

```diff
@@ -86,9 +89,33 @@ export class CollectionDetail extends LiteElement {
         <h1
           class="flex-1 min-w-0 text-xl font-semibold leading-7 truncate mb-2 md:mb-0"
         >
+          ${when(
+            this.collection?.isPublic,
+            () => html`
+              <sl-icon
+                style="font-size: 16px; color: var(--sl-color-success-600)"
+                name="globe2"
+                slot="prefix"
+                title="${msg("Publicly Accessible")}"
+              ></sl-icon>
+            `
+          )}
           ${this.collection?.name ||
           html`<sl-skeleton class="w-96"></sl-skeleton>`}
         </h1>
+        ${when(
+          this.collection?.isPublic,
+          () => html`
+            <sl-button
+              size="small"
+              class="p-2"
+              @click=${() => (this.showEmbedInfo = true)}
+            >
+              <sl-icon name="code-slash"></sl-icon>
+              View Embed Code
+            </sl-button>
+          `
+        )}
```

Comment:
```suggestion
      <div class="flex items-center gap-2 w-full mb-2 md:mb-0">
      ${when(
        this.collection?.isPublic,
        () => html`
          <sl-tooltip content=${msg("Publicly Accessible")}>
            <sl-icon
              style="font-size: 16px; color: var(--sl-color-success-600)"
              name="globe2"
              label="${msg("Publicly Accessible")}"
            ></sl-icon>
          </sl-tooltip>
        `
      )}
        <h1
          class="flex-1 min-w-0 text-xl font-semibold leading-7 truncate"
        >
          ${this.collection?.name ||
          html`<sl-skeleton class="w-96"></sl-skeleton>`}
        </h1>
        </div>
        ${when(
          this.collection?.isPublic,
          () => html`
            <sl-button
              size="small"
              @click=${() => (this.showEmbedInfo = true)}
            >
              <sl-icon name="code-slash"></sl-icon>
              View Embed Code
            </sl-button>
          `
        )}
```

Reply:
Don't want to deal with this via a suggestion, bad diff, should be pre-commit formatted.  Let me know what workflow is preferable here.